pgpool2-4.6.yaml

```
@@ -0,0 +1,259 @@
+      runtime:
+        - busybox
+        - coreutils
+        - merged-usrsbin
+        - openldap
+        - postgresql-client
+        - wolfi-baselayout
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: pgpool
+          version-path: 4/debian-12
+      # copy the patch file to patch the output form bitnami/compat and apply it
+      - runs: |
+          cp bitnami-compat-remove-os-flavour.patch "${{targets.contextdir}}/"
+      - uses: patch
+        working-directory: "${{targets.contextdir}}"
+        with:
+          patches: bitnami-compat-remove-os-flavour.patch
+      # remove patch file to prevent it from being packaged
+      - runs: |
+          rm "${{targets.contextdir}}/bitnami-compat-remove-os-flavour.patch"
+      - name: "Create Symlinks"
+        runs: |
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/pgpool/bin"
+          # symlink all binaries to /opt/bitnami/pgpool/bin
+          for bin in \
+            pcp_attach_node pcp_detach_node pcp_health_check_stats pcp_node_count \
+            pcp_node_info pcp_pool_status pcp_proc_count pcp_proc_info pcp_promote_node \
+            pcp_recovery_node pcp_stop_pgpool pcp_watchdog_info \
+            pg_enc pg_md5 pgpool pgpool_setup pgproto watchdog_setup wd_cli \
+          ; do
+            ln -s "/usr/bin/${bin}" "${{targets.contextdir}}/opt/bitnami/pgpool/bin/${bin}"
+          done
+
+          # copy etc to pgpool/etc
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/pgpool/etc"
+          cp -pr "${{targets.destdir}}/etc/pgpool2/." "${{targets.contextdir}}/opt/bitnami/pgpool/etc/"
+
+          # copy etc to pgpool/etc.default
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/pgpool/etc.default"
+          cp -pr "${{targets.destdir}}/etc/pgpool2/." "${{targets.contextdir}}/opt/bitnami/pgpool/etc.default/"
+
+          # link postgres binaries to /opt/bitnami/postgresql/bin
+          mkdir -p "${{targets.contextdir}}/opt/bitnami/postgresql/bin"
+          for bin in \
+            pg_dump pg_dumpall pg_restore psql \
+          ; do
+            ln -s "/usr/bin/${bin}" "${{targets.contextdir}}/opt/bitnami/postgresql/bin/${bin}"
+          done
+    test:
+      environment:
+        accounts:
+          users:
+            - username: bitnami
+              uid: 1001
+              gid: 0
+          run-as: 0
+        contents:
+          packages:
+            - ${{package.name}}
+            - busybox
+            - coreutils
+            - sudo
+      pipeline:
+        - runs: |
+            test -f /opt/bitnami/pgpool/bin/pgpool
+            test -f /opt/bitnami/postgresql/bin/psql
+        - uses: bitnami/validate-welcome-message
+          with:
+            app-name: pgpool
+
+update:
+  enabled: true
+  ignore-regex-patterns:
+    - _RPM
+  version-transform:
+    - match: "_"
+      replace: .
+  github:
+    identifier: pgpool/pgpool2
+    strip-prefix: V
+    use-tag: true
```

Since this is a version stream for `4.6`, you should add `tag-filter-prefix:` to ensure the automation only picks updates relevant to this version stream:
```suggestion
  github:
    identifier: pgpool/pgpool2
    strip-prefix: V
    use-tag: true
    tag-filter-prefix: V4_6
```
This helps avoid updates from other major versions (e.g., `V4_7`) being incorrectly applied.